Automattic/newspack-plugin
# feat: remove AMP as a required plugin

### All Submissions:






### Changes proposed in this Pull Request:



With the move away from AMP, the AMP plugin should not be required nor installed during the setup anymore. 

### How to test the changes in this Pull Request:

1. Set up Newspack on a fresh site
2. Observe the AMP plugin was not installed
3. Visit the Health Check wizard, observe AMP is not listed anywhere
4. Visit the Configuration tab of the wizard, observe there is not item related to AMP

### Other information:
